### DISCUSSION TITLE
Better voice chat

### DISCUSSION THREAD

**Discussion by oomfiee** [stoatchat/discussions#154] [Feature Suggestions]
here is a mockup for how should the voice chat should look like: ![mockup-1](https://user-images.githubusercontent.com/80048740/134503695-a9c893b7-2815-419f-8a93-5c9bf57c16eb.png)

**Comment by second2050** [comment]
in my opinion if the voice chat participant aren't listed as a list but as icons, they should be inline with the voice channel button itself

**Comment by natejgardner** [comment]
Even just a number and a users icon would be a great start to see a channel is active, maybe a color change too

**Comment by Ashe** [comment]
Make it configurable, I imagine some people will want a list of names, or a grid of just icons, or a summary.
